.env.test

```diff
@@ -9,7 +9,14 @@ OPENSTACK_PASSWORD='pass'
 OPENSTACK_TENANT='test-tenant'
 OPENSTACK_AUTH_URL='http://example.com/auth/url'
 OPENSTACK_REGION='test-region'
-DEFAULT_INSTANCE_BASE_DOMAIN='example.com'
+
+# Since the DEFAULT_INSTANCE_BASE_DOMAIN is used for defining the help
+# text of the BetaTestApplication as well, we need to set this value to
+# our production hosting url. Although this sounds risky, since the API
+# key of Gandi is fake, we cannot modify anything on the production records
+# using unit tests.
+DEFAULT_INSTANCE_BASE_DOMAIN='opencraft.hosting'
```

Comment:
@gabor-boros 

> Since the DEFAULT_INSTANCE_BASE_DOMAIN is used for defining the help text of the BetaTestApplication as well,

Why is this a problem? What's wrong with it defaulting to example.com?

Reply:
Fair point, I wrote that comment before I hardcoded the help text of the model field. I'll adjust the code accordingly.

Reply:
Resolving this conversation since https://github.com/open-craft/opencraft/pull/643/commits/7444bddb2d8a9d66fac7880e4d2ba9c2ae382917 is committed